src/main/pages/scenario.py

```diff
@@ -0,0 +1,67 @@
+from django.db import models
```

Comment:
I'm not sure why this model and related pages are in this branch and how to relate this to the design